Formula/liquibase.rb

```diff
@@ -1,7 +1,7 @@
 class Liquibase < Formula
   desc "Library for database change tracking"
   homepage "https://www.liquibase.org/"
-  url "https://github.com/liquibase/liquibase/releases/download/liquibase-parent-3.8.0/liquibase-3.8.0-bin.tar.gz"
+  url "https://github.com/liquibase/liquibase/releases/download/liquibase-parent-3.8.0/liquibase-3.8.4-bin.tar.gz"
```

Comment:
This URL doesn't exist

Reply:
I corrected the link: https://github.com/liquibase/liquibase/releases/download/v3.8.4/liquibase-3.8.4.tar.gz

Reply:
It is required to update SHA256 accordingly

Reply:
I updated the SHA256 accordingly.  Thanks for your help!